### ISSUE TITLE
Test: accessibilityInformation API for TreeItem, StatusBarItem and TimelineItem

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#100936]
Refs: https://github.com/microsoft/vscode/issues/95360

- [x] anyOS @JacksonKearl
- [x] anyOS @sbatten

Complexity: 4

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23100936%0A%0A)

---

We introduced the `accessibilityInformation` API to `TreeItem`, `StatusBarItem` and `TimelineItem`.

* Write an extension that adds two out of those three. For example a `TreeItem` and a `StatusBarItem`. You can use the [tree view sample as a starter](https://github.com/microsoft/vscode-extension-samples/tree/master/tree-view-sample) and the [status bar sample as another starter](https://github.com/microsoft/vscode-extension-samples/tree/master/statusbar-sample)
* In your extension make sure that for `TreeItem`, `StatusBarItem` or `TimelineItem` you return the new `AccessibilityInformation` object
* Run your extension, turn on a screen reader and make sure the `label` which you provided in the `AccessibilityInformation` is properly read out once the focus is put on your contributed item
* Testing out the `role` is a bit more tricky with basic screen reader knowledge, but you can for example return a role `checkbox` and the screen reader should announce that your item is a checkbox. For other roles you can check [this](https://w3c.github.io/aria/#widget_roles)
* Contribute a status bar item which does not have `AccessibilityInformation` set, and make sure to use a codicon in your status bar item. Turn on a screen reader and verify that the readable name is extracted from a codicon and that is read.

Note: for focusing status bar items you can press `F6` a couple of times until the focus is on the status bar and you can use `left / right` to navigate through the items.

**Comment by sbatten** [comment]
I had no issues with the TreeItem, but for the StatusBarItem, the sample just stops working if I add this line: `myStatusBarItem.accessibilityInformation = { label: 'An Accessible label' };`

I don't see any errors, it just never shows the status bar item

**Comment by isidorn** [comment]
@sbatten please note that the simply is a bit funky and it only show the status bar if you have multiple lines selected. If you have a multi line selection do you see it then?
If not, do you see the status bar if you remove the `accessibilityInformation`.
I tried it out and for me it works just fine.

**Comment by sbatten** [comment]
![recording (6)](https://user-images.githubusercontent.com/6561887/86266537-36eb0e80-bb7a-11ea-8088-f4a75c737315.gif)
